libs/island-ui/core/src/lib/CategoryCard/CategoryCard.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { Box } from '../Box/Box'
+import { FocusableBox } from '../FocusableBox/FocusableBox'
+import { Inline } from '../Inline/Inline'
+import { Tag } from '../Tag/Tag'
+import { Text } from '../Text/Text'
+
+type Tag = {
+  label: string
+  href?: string
+  onClick?: () => void
+}
+
+type CategoryCardProps = {
+  heading: string
+  text: string
+  tags?: Tag[]
+  href?: string
+  colorScheme?: 'blue' | 'red'
```

Comment:
should this be more descriptive towards the main usage, like 'primary', 'secondary' or anything that would make more sense regarding how this should be used?

Reply:
When we know exactly how this will be used we should do that. Right now we only know of 2 projects that will use this component and each of these projects wants a different color, hence the two available ones.

Reply:
![image](https://user-images.githubusercontent.com/8494120/96599194-315b3500-12df-11eb-8c1d-c57343daf335.png)

![image](https://user-images.githubusercontent.com/8494120/96599241-3f10ba80-12df-11eb-808b-e735e4396b9b.png)